client/src/pages/NrMap.js

```diff
@@ -33,61 +33,75 @@ const Cards = styled.div`
   margin: 10px;
 `;
 
-class NrMap extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      latitude: 52.520008,
-      longitude: 13.404954,
-      rec: [],
-      plcDtl: [],
-    };
-    this.googleMapRef = React.createRef();
-  }
-  componentDidMount() {
+const map = {
+  googleMap: null,
+  marker: null,
+  place: null,
+};
+
+const NrMap = () => {
+  const [coordinates, setCoordinates] = useState({
+    latitude: 52.520008,
+    longitude: 13.404954,
+  });
+  // rec-> list of hospitals according to nearbySearch
```

Comment:
Instead of comments, what do you think about renaming `rec` and `plcDtl ` for better semantics?

Reply:
done

Reply:
done